### PR TITLE
Replace Jetty with Tomcat

### PR DESCRIPTION
**Pull request by kunklejr**
- [x] @joeferner
- [x] @diegogrz
- [x] @mwizeman @sfeng88
- [ ] @joeybrk372 @rygim @jharwig @EvanOxfeld

Merge after https://github.com/v5analytics/visallo-lts/pull/2256

Testing Instructions:
Follow the instructions in the Getting Started guide of the documentation to verify the server starts when running the documented Maven command.

Points of Regression:

CHANGELOG
Changed: Replaced Jetty with Tomcat as the only supported web server.
Removed: Removed the Jetty server

### CODE REVIEW

File: config/visallo.properties [hunk 1]
```diff
@@ -79,8 +79,7 @@ security.visibilityTranslator=org.visallo.core.security.DirectVisibilityTranslat
 # Uncomment to allow plain HTTP. HTTPS is forced otherwise.
 #http.transportGuarantee=NONE
 
-# Enable/Disable Visallo Gzip filter (on Jetty the default is true, other containers default to false)
-#  The Visallo Gzip filter is known to have issues on Tomcat containers.
+# Enable/Disable Visallo Gzip filter. The Visallo Gzip filter is known to have issues on Tomcat containers. (default is false)
```
Comment (EvanOxfeld) [inline config/visallo.properties]: Should this be removed?

Comment (jharwig) [inline config/visallo.properties]: 👍 there might be some enterprise doc changes too

File: web/web-base/pom.xml [hunk 2]
```diff
@@ -21,21 +21,11 @@
...
             <artifactId>visallo-closure-compiler</artifactId>
         </dependency>
         <dependency>
             <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-server</artifactId>
-            <scope>provided</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-servlets</artifactId>
```
Comment (jharwig) [inline web/web-base/pom.xml]: Still need jetty-servlets?

File: config/visallo.properties (hunk 1 above)
Comment (kunklejr) [inline config/visallo.properties]: I wasn't quite sure if we should remove that or not, and wanted to take more time to look into the gzip issues as a separate PR.

File: web/web-base/pom.xml (hunk 2 above)
Comment (kunklejr) [inline web/web-base/pom.xml]: jetty-servlets provides our gzip filter

**Pull request merged**